department-of-veterans-affairs/caseflow
# Added Caregiver as an issue category

Resolves [CASEFLOW-2074](https://vajira.max.gov/browse/CASEFLOW-2074)

### Description
Added "Caregiver" to the list of categories.

### Acceptance Criteria

### Testing Plan

1. Navigate to test/users
2. Search for “bva intake” and log in as any intake user, such as BVADWISE
3. Select the Mail Intake tab and select “Start”
4. Select the form as Board Appeal (VA Form 10182) and continue to search
5. Enter “872958715” as the Veteran’s ID and search
6. See more file numbers here (this is not an exhaustive list): bgs_setup.csv
7. Add a date (after February 19, 2019) and select any docket type, and select any other options
8. Click “Continue to next step” and ensure that you can proceed
9. Click “Add issue”
10. If a modal appears showing a list of issues, click “None of these match” to go to the next modal. If a modal appears asking for a benefit type and category, proceed with the current modal.
11. Select “VHA” as the benefit type
12. Select an issue category, enter a decision date between a year ago and the receipt date, and a description
13. Continue to next step. If you get an untimeliness error, you can cancel or enter a timeliness exemption. If you get a legacy opt in modal, you can select the “None of these match” radio button and proceed.
14. Click “Establish Claim”
15. You should now be able to search for the appeal using the veteran’s file number on the /search page or see the appeal in VHA’s queue at /decision_reviews/vha

### User Facing Changes

 BEFORE|AFTER
 
![image](https://user-images.githubusercontent.com/69862429/128072000-323f52d5-6e3d-42c4-af3e-709bee78dd3a.png)
|
![image](https://user-images.githubusercontent.com/69862429/128070728-6705f4a5-5a35-4206-8971-f13cdb047e0e.png)
